spec/lib/models/node-spec.js

```diff
@@ -152,5 +152,72 @@ describe('Models.Node', function () {
             });
         });
 
+        describe('tags', function() {
+            before(function() {
+                this.subject = this.attributes.tags;
+            });
+
+            it('should be an array', function() {
+                expect(this.subject.type).to.equal('array');
+            });
+
+            it('should default to empty', function() {
+                expect(this.subject.defaultsTo).to.deep.equal([]);
+            });
+        });
+
+        describe('Node Tag Functions', function() {
+            var collection = {
+                update: sinon.stub().resolves()
+            };
+
+            before(function() {
+                sinon.stub(this.model, "native", function(cb) { cb(null, collection); });
```

Comment:
It is stubbed out with this pattern so the underlying code can promisify the function.  It is being stubbed out to remove the mongodb dependency from the unittest. Functional/integration testing would be used to validate MongoDB behavior/interactions, not unittests.